### DISCUSSION TITLE
LR1121 - VTCXO?

### DISCUSSION THREAD

**Discussion by krdarrah** [ExpressLRS/ExpressLRS#3045] [Q&A]
Hello - does anyone know why the LR1121.cpp does not have code to enable and set the VCTXO?  I thought this is required if using a TCXO?  and I see a lot of boards out there using these oscillators, but I guess they're just powering them from the same VBAT 3.3V?  I took a screen shot from the user manual:
![Screenshot 2024-11-20 at 5 30 42 PM](https://github.com/user-attachments/assets/f3869493-a113-4174-9734-727073dba875)
ual:

**Comment by pkendall64** [comment]
We just haven't got round to adding it as most manufactured devices directly connect the TCXO to a seperate power source.

**Reply by krdarrah** [reply]
thanks - do you think it's coming?  I'll prep a board I'm working on for a customer for that with some zero ohm jumps

**Reply by pkendall64** [reply]
It's not hard to add, it's just theres quite a bit of cleanup going on at the moment to get ready for more V4 features. This might be able to make it into V4.
